src/util/string.h

```diff
@@ -212,14 +212,12 @@ inline std::string MakeUnorderedList(const std::vector<std::string>& items)
 }
 
 /**
- * Check if a string does not contain any embedded NUL (\0) characters
+ * Check if a string contains any embedded NUL (\0) characters
```

Comment:
the comment just repeats what the code already states clearly, I'd just remove it

Reply:
Given that you forgot to update it, I'd say it doesn't have a big added value, let's just delete it.

Reply:
I'd like to see another concept ACK to removing the comment I think before I do so, if someone else thinks it's valuable to drop the comment, I will.